db/schema.rb

```diff
@@ -18,21 +17,20 @@
 
   create_table "admins", force: :cascade do |t|
     t.string "css_id", null: false
+    t.index ["css_id"], name: "index_admins_on_css_id", unique: true
   end
 
-  add_index "admins", ["css_id"], name: "index_admins_on_css_id", unique: true, using: :btree
```

Comment:
This was an automated change by rails update (ref. https://apidock.com/rails/ActiveRecord/ConnectionAdapters/Table/index)